src/main/kotlin/com/pubnub/api/PNConfiguration.kt

```diff
@@ -27,15 +27,23 @@ open class PNConfiguration(
 ) {
 
     init {
-        PubNubUtil.require(uuid.isValid(), PubNubError.UUID_NULL_OR_EMPTY)
+        PubNubUtil.require(uuid.isValid(), PubNubError.USERID_NULL_OR_EMPTY)
     }
 
+    constructor(userId: UserId, enableSubscribeBeta: Boolean = false) : this(userId.value, enableSubscribeBeta)
```

Comment:
Should we deprecate constructor with `uuid` together with a `uuid` setter?

Reply:
I like this idea. I will add deprecation code.